eng/targetingpacks.targets

```diff
@@ -147,6 +147,17 @@
     </ItemGroup>
   </Target>
 
+  <!-- Use local Crossgen2 pack for NetCoreAppCurrent. -->
+  <Target Name="UpdateLocalCrossgen2Pack"
+          Condition="'$(UseLocalCrossgen2Pack)' == 'true'"
+          AfterTargets="ResolveFrameworkReferences">
+    <ItemGroup>
+      <ResolvedCrossgen2Pack PackageDirectory="$(CoreCLRArtifactsPath)\$(BuildArchitecture)\crossgen2\"
+                             Condition="$([System.String]::new('%(Identity)').StartsWith('$(LocalFrameworkOverrideName).Crossgen2'))" />
```

Comment:
Are you sure this is necessary? This item should only contain Crossgen2 framework pack items.